web/concrete/core/libraries/view.php

```diff
@@ -992,4 +992,13 @@ public function render($view, $args = null) {
 			}
 			
 		}
+
+		/**
+		 *
+		 * @param $file string Full file path as string
+		 */
+		public function includeFileInScope($file, $args = array()) {
+			extract($args);
```

Comment:
Thanks Korvin! I've fixed this here https://github.com/concrete5/concrete5-legacy/pull/1927/files#diff-ef7d3a5d223a0015b1bb927dc2fc16e3R1019